README.md

```diff
@@ -211,6 +212,37 @@ Please note that **the message update step does not accept a channel name.** Set
     SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
 ```
 
+#### Reply to a message
+
+If you want to post a message as a reply, you can populate the `payload` with a `"thread_ts"` field. The steps after the first message posting can have `"thread_ts": "${{ steps.slack.outputs.ts }}"` in their payload. In order to reply to an existing message.
+
+Please note that **that reply to a message does not accept a channel name.** Set a channel ID for the actions that reply to messages in thread.
+
+```yaml
+- id: slack
```

Comment:
Let's give this a more descriptive ID so the association between the step ID and the use of that ID to get its output in a later step is clearer. Maybe:

```suggestion
- id: deployment_message
```

... and then using `"thread_ts": "${{ steps.deployment_message.outputs.ts }}"` later in the example.